src/main/java/org/jpeek/App.java

```diff
@@ -76,6 +76,10 @@
  *  Once they are resolved, cover the metric with autotests and add it
  *  to reports list.
  *  (details on how to test the metrics are to be negotiated here - #107)
+ *
+ * @todo #64:30min C3 metric has impediments (see puzzles in C3.xls).
```

Comment:
@rok-povsic Please, remove this puzzle, it is too early for it. It will be turned into a task and it will wait a lot of time to be resolved. I asked other contributors to add such puzzles only when there were little changes needed in order for the metric to work.